test/d_do_test.d

```diff
@@ -565,6 +573,25 @@ int tryMain(string[] args)
     if (!gatherTestParameters(testArgs, input_dir, input_file, envData))
         return 0;
 
+    // Clear the DFLAGS environment variable if it was specified in the test file
+    if (testArgs.dflags !is null)
+    {
+        if (testArgs.dflags != "")
+            throw new Exception("The DFLAGS test argument must be empty: It is '" ~ testArgs.dflags ~ "'");
+
+        // `environment["DFLAGS"] = "";` doesn't seem to work on Win32 (might be a bug
+        // in std.process). So, resorting to `putenv` in snn.lib
+        version(Win32)
+        {
+            char[] varExpression = cast(char[])"DFLAGS=\0";
+            putenv(varExpression.ptr);
```

Comment:
Declaring `extern(C) int putenv(const char *);` globally and calling `putenv("DFLAGS=");` worked for me. The literal already has the trailing zero and converts implicitly.

Reply:
Done.